.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.po
```

Comment:
Yeah, you are right. Having `.mo` files  actually could breaks  rebase for Viewflow pro repository.